<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetZdroj</class>
 <widget class="QWidget" name="widgetZdroj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zdroj</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="indentation">
      <number>10</number>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>NÃ¡zev</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>cmd</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hodnota</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>suffix</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="butStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="butStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
